content/_about.md

```diff
@@ -6,4 +6,20 @@ url = "about"
 
 ## About
 
-This site is
+Uruguay has great talent doing some awesome work but we fail miserably communicating that "awesomeness", most of the developers fail communicating their achievements, while others don't really care.
+
+We built `*.js.uy` as a way to list and promote all the great things we are doing as a JS community in Uruguay.
+
+We think that getting noticed and spreading the word of what we do would have a positive impact on the developers life by: reaching better clients, raising salaries and allowing people to work on projects they care about.
+
+To summarize, our goals are:
+
+* Help people to better market themselves
+* Get noticed at an international level
+* Put Uruguay in the map for the global JS community
```

Comment:
The goal should not be restricted to better global "awareness". I think it should push hard to improve collaboration and lower segregation among the Uruguayan's developers. 
Strictly speaking, global is a superset of local but in reality I don't see how you can get good ideas spread globally if they are unknown to your peers. 

In summary, I'll remove goal 2( almost the same than 3) in favor of something in the lines of:
* Boost collaboration among the Uruguay-based developers